give-a-talk.md

```diff
@@ -1,40 +1,46 @@
 ---
 layout: page
-title: Give a Lightning Talk
+title: We want you to give a talk!
 ---
 {% include JB/setup %}
 
-### We want you to give a talk!
+If you have something JavaScript related to share with your peers that you could say/show in less than 5 minutes, **we want to hear you give a talk**!
 
-Have you been learning JavaScript and want to spend a couple minutes sharing something you learned?
+Have you been learning JavaScript and want to spend a couple minutes sharing something you learned? Did you find a library or discover a JS feature you'd like to share? Have an interesting method for learning or teaching JavaScript? Want to share something you made? Want to show off a niche feature of one of your favorite libraries?
 
-Did you find a library or discover a JS feature you'd like to share?
+We also have a wiki page with [talk ideas][1] that you can use for inspiration.
 
-Have an interesting method for learning or teaching JavaScript?
+---
 
-Want to share something you made?
+__*If you are unsure what to talk about do not hesitate to jump onto one of the [chat channels][2] and ask about presenting.*__
```

Comment:
@treyhunner adding a chat callout for people who are not sure what to talk about.